docs/userGuide/syntax/code.md

```diff
@@ -228,6 +228,17 @@ or the java code `public static void main(String[] args)`{.java}.
 </variable>
 </include>
 
+{% raw %}
+
+##### Displaying content within curly braces: `{{ content }}`
+
+If your code contains special Nunjucks tags like `{{` or `}}`,use a [raw-endraw block](../tipsAndTricks.html#using-raw-endraw-to-display-content):
```

Comment:
```suggestion
If your code contains special Nunjucks tags like `{{` or `}}`, use a [raw-endraw block](../tipsAndTricks.html#using-raw-endraw-to-display-content):
```

Reply:
Fixed in [0b0701a](https://github.com/MarkBind/markbind/pull/2118/commits/0b0701af78d7a1071712a97fa26c0c3803d50867), along with the same error in `mathformulae.md` in the same directory!